src/main/webapp/js/user-page-loader.js

```diff
@@ -35,66 +35,66 @@ const parameterUsername = urlParams.get('user');
 
 // URL must include ?user=XYZ parameter. If not, redirect to homepage.
 if (!parameterUsername) {
-  window.location.replace('/');
+    window.location.replace('/');
 }
 
 /** Sets the page title based on the URL parameter username. */
 function setPageTitle() {
-  document.getElementById('page-title').innerText = parameterUsername;
-  document.title = parameterUsername + ' - User Page';
+    document.getElementById('page-title').innerText = parameterUsername;
+    document.title = parameterUsername + ' - User Page';
 }
 
 /**
  * Shows the message form if the user is logged in and viewing their own page
  * Hides private messaging option if user is on private messaging page
  */
- function showMessageFormIfLoggedIn() {
-   fetch('/login-status')
-       .then((response) => {
-         return response.json();
-       })
-       .then((loginStatus) => {
-         if (loginStatus.isLoggedIn) {
-           const messageForm = document.getElementById('message-form');
-           messageForm.action = '/messages?recipient=' + parameterUsername;
-           messageForm.classList.remove('hidden');
-           
-           /** Using 34 because @codestudents.com is 17 characters long
-            * and there's at least 2 people in a direct message
-            */
-           if (parameterUsername.length < 34) {
-             const privateOption = document.getElementById('private-option');
-             privateOption.classList.remove('hidden');
-
-             const sendOption = document.getElementById('send-option');
-             sendOption.classList.remove('hidden');
-          } else {
-             document.getElementById('private-option-checkbox').checked = true;
-          }
-           document.getElementById('about-me-form').classList.remove('hidden');
-         }
-       });
-
- }
+function showMessageFormIfLoggedIn() {
```

Comment:
Let's avoid formatting other people's code unless you've made a change.

Reply:
Ok, my error